nixos/modules/config/fonts/fontconfig.nix

```diff
@@ -344,6 +346,21 @@ in
               in case multiple languages must be supported.
             '';
           };
+
+          emoji = mkOption {
+            type = types.listOf types.str;
+            default = ["Noto Color Emoji"];
```

Comment:
Do we actually need to do this?

Reply:
With the current fontconfig, JoyPixels is 1st in line.

Reply:
Yes. With the current fontconfig, Noto is not very high in the priority list.

Reply:
So let's say the newer fontconfig gets merged, will the priority in the default config be higher but still lower than this? If so I'll make note of this in the issue tracker.

Reply:
The order is listed here https://gitlab.freedesktop.org/fontconfig/fontconfig/commit/c41c9220181b203d1cf1f6435f6e3735cb7c84ac